### PR TITLE
Cli stats server

### LINKED ISSUE #6080: CLI: `stats_server`

### PR DESCRIPTION
**Pull request by TimeEngineer**
closes #6080

### CODE REVIEW

File: cli/src/stats_server.rs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    addr: BackendAddr,
+    /// Administration token
+    #[arg(short, long)]
+    token: String,
+    /// Output format (json/csv)
+    #[arg(short, long, default_value_t = Format::Json)]
+    format: Format,
+    /// Ignore everything after this date (e.g: 2024-01-01T00:00:00-00:00)
+    #[arg(long)]
+    at: Option<DateTime>,
```
Comment (FirelightFlagboy) [inline cli/src/stats_server.rs:22]: I would prefer `start_date` as the doc-comment say it ignore everything before that date.

`at` could confuse the user that the stat will only be for a specific date

File: cli/src/stats_server.rs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+impl std::fmt::Display for Format {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::Json => "json",
+                Self::Csv => "csv",
+            }
+        )
```
Comment (FirelightFlagboy) [inline cli/src/stats_server.rs:52]:
```suggestion
        f.write_str(match self {
            Self::Json => "json",
            Self::Csv => "csv",
        })
```

File: cli/src/stats_server.rs (hunk 1 above)
Comment (TimeEngineer) [reply cli/src/stats_server.rs:22]: Not before, after

Comment (TimeEngineer) [reply cli/src/stats_server.rs:22]: Maybe you prefer `ignore-after` ?

Comment (FirelightFlagboy) [reply cli/src/stats_server.rs]: Oops, `end-date` then

Comment (TimeEngineer) [reply cli/src/stats_server.rs]: Done

**Pull request merged**